templates/car.html

```diff
@@ -1,40 +1,71 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+</head>
+<body style="background-color: #e5e5e5;">
+    <div>
+        <h1 class="header">Add Car Information</h1>
+    
+        <label for="vin">VIN</label>
+        <input id="vin" type="text" />
```

Comment:
for vin, I think the real world constraints are that it has to be between 11-17 characters long. So add a minlength and maxlength
```suggestion
        <input id="vin" type="text" minlength="11" maxlength="17"/>
```